### ISSUE TITLE
Depth360: Monocular Depth Estimation using Learnable Axisymmetric Camera Model for Spherical Camera Image. (arXiv:2110.10415v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#12306]
Depth360: Monocular Depth Estimation using Learnable Axisymmetric Camera Model for Spherical Camera Image. (arXiv:2110.10415v1 [cs.CV])<br>
https://ift.tt/3nexFqp<br>
<p>Self-supervised monocular depth estimation has been widely investigated to estimate depth images and relative poses from RGB images. This framework is attractive for researchers because the depth and pose networks can be trained from just time sequence images without the need for the ground truth depth and poses.</p>
<p>In this work, we estimate the depth around a robot (360 degree view) using time sequence spherical camera images, from a camera whose parameters are unknown. We propose a learnable axisymmetric camera model which accepts distorted spherical camera images with two fisheye camera images. In addition, we trained our models with a photo-realistic simulator to generate ground truth depth images to provide supervision. Moreover, we introduced loss functions to provide floor constraints to reduce artifacts that can result from reflective floor surfaces. We demonstrate the efficacy of our method using the spherical camera images from the GO Stanford dataset and pinhole camera images from the KITTI dataset to compare our method's performance with that of baseline method in learning the camera parameters.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
